Packs/Core/Playbooks/playbook-Ransomware_Response.yml

```diff
@@ -1055,6 +1082,17 @@ tasks:
             value:
               simple: unTrusted
           ignorecase: true
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: Core.AnalyticsPrevalence.Hash
+                accessor: value
+            iscontext: true
+          right:
+            value:
+              simple: "true"
```

Comment:
shouldn't it be false?

Reply:
oh I saw just saw its for the false positive condition. I think we shouldnt add the prevalence check there.